apps/web/src/components/Disclosure.tsx

```diff
@@ -15,10 +15,12 @@ export const Disclosure: React.FC<DisclosureProps> = ({ buttonText, content }) =
         <>
           <HeadlessDisclosure.Button className={'flex justify-between items-center w-full'}>
             {buttonText}
-            <FontAwesomeIcon
-              icon={faChevronDown}
-              className={classnames('text-gray-500 mr-5', { 'transform rotate-180': open })}
-            />
+            <div style={{ width: '50px' }}>
+              <FontAwesomeIcon
+                icon={faChevronDown}
+                className={classnames('text-gray-500 mr-5', { 'transform rotate-180': open })}
```

Comment:
Same thing, we should almost never need inline styles 
```suggestion
                className={classnames('text-gray-500 mr-5 h-5', { 'transform rotate-180': open })}
```

Reply:
Something funky is going on with tailwind. Your suggestion works, but when I translate it to the div above, it doesn't work. It seems 'h-5' != width:'50px'. 

Thats why I went with style, because it didn't work initially with tailwind. 
<img width="1340" alt="Screen Shot 2022-01-12 at 1 03 58 PM" src="https://user-images.githubusercontent.com/54554766/149221233-9d267dfe-2535-43e4-bbb3-5845c16e44aa.png">

Reply:
Looks like it's because the style is applied to a wrapper div instead of the icon itself

Reply:
h-5 is 1.25rem not 50px